distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -1,3 +1,230 @@
 
-// https://codice.atlassian.net/browse/DDF-2651
-placeholder text
+Introduction: The ${catalog-ui} represents the most advanced search interface available with ${branding}.
+It provides a metadata search and discovery, resource retrieval, and workspace management with a 3d or optional 2d map visualization.
+
+[NOTE]
+====
+For more detail on any feature or button within the Catalog UI, click the `?` button in the upper right of the screen; then, hover over any item on the screen and a contextual tooltip will to define its purpose.
+To exit this mode, click the `?` again or press `escape`.
+====
+
+=== Accessing ${catalog-ui}
+
+The default location for the ${catalog-ui} is ${secure_url}/search/catalog
+
+=== Catalog UI Guest Users
+
+Guest users not signed in to ${branding} will have access to search functions, but all workspace configuration and settings will only exist locally and not be available for sharing.
+
+=== Workspaces in ${catalog-ui}
+
+Within the ${catalog-ui}, workspaces are collections of settings and standing searches that can be shared between users and stored to be accessed repeatedly.
```

Comment:
Recommend "...collections of settings, searches, and bookmarks...".

We don't use the term "standing searches" in the UI.  Rather there are searches with notifications.